doc/whatsnew/2.0.rst

```diff
@@ -89,6 +89,51 @@ New checkers
           def not_useless_1(self, first, *args):
               return super(Impl, self).not_useless_1(first + some_value, *args)
 
+* A new warning was added, ``len-as-condition``, which is used whenever
+  we detect that a condition uses ``len(SEQUENCE)`` incorrectly. Instead
+  one could use ``if SEQUENCE`` or ``if not SEQUENCE``.
+
+  For instance, all of the examples below are incorrect:
```

Comment:
I would not use incorrect here, rather something along the lines "all the examples below could be written in a more natural way".